src/boardlib/__main__.py

```diff
@@ -1,17 +1,16 @@
 import argparse
```

Comment:
There are a lot of changes in this file, but it seems to me like at most this should just be a one-line change to add `aggregate=False`, but really there shouldn't be any changes at all because aggregate is already defaulting to False. Can we revert this file please? There are multiple regressions, e.g. to the command output, the imports, and the styling.